extras/minimal-printf/README.rst

```diff
@@ -0,0 +1,51 @@
+======================================
+Minimal printf replacement for Duktape
+======================================
+
+The ``duk_minimal_printf.c`` provides a portable provider for ``printf()``
+with a feature set matching minimally what Duktape needs.  The provider
+compiles to less than 1kB.
+
+Assumptions:
+
+* ``sizeof(void *) <= sizeof(long)``
```

Comment:
Note that this assumption is broken on, e.g. Windows x64 where `sizeof(void*) == 8` but `sizeof(long) == 4` (!)